scripts/collect-assets.js

```diff
@@ -106,16 +107,20 @@ const actions = [
     }),
   }),
 
-  // Extract seed rulesets from the linux-x64_64 downloaded workflow artifact or release asset
+  // Download and extract seed rulesets from a rulesets repo release
   async () => ({
     id: "seed rulesets",
-    meta: await unpackAssets({
-      title: "rulesets",
-      sourceDirectory: join(DOWNLOAD_CACHE, "assets"),
-      targetDirectory: () => join(DOWNLOAD_DIR, "rulesets"),
+    meta: await downloadAndExtractGitHubReleaseSourceCode({
+      downloadDirectory: join(DOWNLOAD_CACHE, "sources"),
+      targetDirectory: join(DOWNLOAD_DIR, "rulesets"),
 
-      globs: ["example/analysis/rulesets/**/*"],
-      assets: [{ name: "kai-rpc-server.linux-x86_64.zip" }],
+      org: "konveyor",
+      repo: "rulesets",
+      releaseTag: "v0.6.1",
+      bearerToken,
+
+      context: "{{root}}/default/generated",
+      globs: ["**/*"],
```

Comment:
This is the entry point for downloading and extrating the GH release sources (the zipball)